[2024-03-02T03:16:38.529+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_pipeline.extract_orders scheduled__2024-03-02T02:01:00+00:00 [queued]>
[2024-03-02T03:16:38.631+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_pipeline.extract_orders scheduled__2024-03-02T02:01:00+00:00 [queued]>
[2024-03-02T03:16:38.632+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-02T03:16:38.923+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_orders> on 2024-03-02 02:01:00+00:00
[2024-03-02T03:16:39.026+0000] {standard_task_runner.py:60} INFO - Started process 1807 to run task
[2024-03-02T03:16:39.122+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_pipeline', 'extract_orders', 'scheduled__2024-03-02T02:01:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/ecom.py', '--cfg-path', '/tmp/tmpdya491qq']
[2024-03-02T03:16:39.151+0000] {standard_task_runner.py:88} INFO - Job 60: Subtask extract_orders
[2024-03-02T03:16:39.853+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-02T03:16:42.337+0000] {task_command.py:423} INFO - Running <TaskInstance: ecommerce_pipeline.extract_orders scheduled__2024-03-02T02:01:00+00:00 [running]> on host 70b88be0f8e4
[2024-03-02T03:16:43.969+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='extract_orders' AIRFLOW_CTX_EXECUTION_DATE='2024-03-02T02:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-02T02:01:00+00:00'
[2024-03-02T03:16:43.993+0000] {logging_mixin.py:188} INFO - Here's some information about the data stored in ./data/orders.csv:
[2024-03-02T03:16:43.996+0000] {logging_mixin.py:188} INFO - 
There are 100 rows and 5 columns in the DataFrame.
[2024-03-02T03:16:43.997+0000] {logging_mixin.py:188} INFO - 
The columns in the DataFrame have the following types: 
[2024-03-02T03:16:44.000+0000] {logging_mixin.py:188} INFO - order_id        int64
customer_id     int64
product_id      int64
quantity        int64
order_date     object
dtype: object
[2024-03-02T03:16:44.001+0000] {logging_mixin.py:188} INFO - 
To view the DataFrame extracted from ./data/orders.csv, display the value returned by this function!
[2024-03-02T03:16:44.002+0000] {python.py:201} INFO - Done. Returned value was:     order_id  customer_id  product_id  quantity  order_date
0          1           14           3         3  2023-01-01
1          2           14           3         3  2023-01-02
2          3           13           4         4  2023-01-03
3          4           14           3         3  2023-01-04
4          5            9           1         3  2023-01-05
..       ...          ...         ...       ...         ...
95        96            4           5         4  2023-04-06
96        97            6           3         2  2023-04-07
97        98            7           1         4  2023-04-08
98        99           12           1         3  2023-04-09
99       100           12           2         2  2023-04-10

[100 rows x 5 columns]
[2024-03-02T03:16:44.180+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ecommerce_pipeline, task_id=extract_orders, execution_date=20240302T020100, start_date=20240302T031638, end_date=20240302T031644
[2024-03-02T03:16:44.562+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-02T03:16:44.634+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
